src/h5web/toolbar/Toolbar.module.css

```diff
@@ -2,7 +2,9 @@
   flex: 1 1 0%;
   display: flex;
   min-width: 0;
+  height: 2.5rem;
   padding: 0 0.5rem;
+  background-color: var(--h5w-toolbar--bgColor, aliceblue);
```

Comment:
I set a height and bg color here, even though we don't need them in H5Web, so that the toolbar can be used on its own in Braggy, without a wrapper element.